cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceTaskContext.java

```diff
@@ -0,0 +1,84 @@
+package co.cask.cdap.api.mapreduce;
+
+/*
+ * Copyright © 2014-2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import co.cask.cdap.api.RuntimeContext;
+import co.cask.cdap.api.ServiceDiscoverer;
+import co.cask.cdap.api.annotation.Beta;
+import co.cask.cdap.api.artifact.PluginContext;
+import co.cask.cdap.api.data.DatasetContext;
+import co.cask.cdap.api.templates.AdapterContext;
+import co.cask.cdap.api.workflow.Workflow;
+import co.cask.cdap.api.workflow.WorkflowToken;
+
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * MapReduce task execution context. This is similar to MapReduceContext, but without ability to configure
```

Comment:
We don't need to say it is similar to MapReduceContext, right? They are not really related.